### PR TITLE
Update database docs diagrams to schema 28

### PR DESCRIPTION
**Pull request by yvanzo**
# Problem

* Diagrams need to be updated for schema 28.
* Diagrams have to be reviewed for:
  * relevance of their descriptions and table lists
  * coverage of the documentation

# Solution

Reviewed in team meeting, made some changes in comments and tooltips.

# Checklist for author

* [x] Get text changes approved
* [x] Generate graphs/images again with `make`
* [x] Reapply aesthetic changes with diff but for `entity_network_details` which might require to go through `inkscape` again.
* [x] Run `make normalized` and commit

# Action

1. Upload the generated files to the wiki (as new revisions for current diagrams).

### CODE REVIEW

File: docs/database_schema_diagrams/source/entity_network_details.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
-  "tooltip": "Tables specifically related to core entities",
-  "_comment": "It shows tables unique to some entities, that is, beyond aliases, collections, edits, relationships, subscriptions, ratings, tags.",
+  "tooltip": "Tables specifically related to some primary entities",
```
Comment (reosarevok) [inline docs/database_schema_diagrams/source/entity_network_details.json:2]: "some primary"? Should it be "all primary" and we can add any missing ones, or?

Comment (yvanzo) [reply docs/database_schema_diagrams/source/entity_network_details.json]: Correct, I dropped “some” and tried to make it more clear with https://github.com/metabrainz/musicbrainz-server/pull/2987/commits/d34e4232a251e0652995c1ce7138e48227438a74#diff-823a23760774550d2454cd9e3f8a20311f20a85cf6672dcdc93cdbc00fc03ae7

File: INSTALL.md [hunk 2]
```diff
@@ -334,6 +334,7 @@ Creating the database
         You can import this sample dump in the same way as the full dump above.
 
     4.  Build materialized tables (optional but recommended)
+        <a name="build-materialized-tables"></a>
```
Comment (reosarevok) [inline INSTALL.md:337]: Is it doable to just anchor to a span around the actual header?

File: docs/database_schema_diagrams/source/entity_network_details.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "tooltip": "Tables specifically related to core entities",
-  "_comment": "It shows tables unique to some entities, that is, beyond alias, collection, edit, link, subscription, rating, tag.",
+  "_comment": "It shows tables unique to some entities, that is, beyond alias, collection, edit, relationship, subscription, rating, tag.",
```
Comment (reosarevok) [inline docs/database_schema_diagrams/source/entity_network_details.json]: The actual table is `link`, but not sure if that matters.

Edit: I see you're moving away from table names on the next commit, so assuming that's a good idea then it does not matter :)

File: docs/database_schema_diagrams/source/cover_art_details.json [hunk 4]
```diff
@@ -1,6 +1,6 @@
 {
   "tooltip": "Tables for cover art",
-  "_comment": "It shows tables used to set a cover art for releases and release groups. These two core entities are highlighted. It also shows when tables belong to the 'cover_art_archive' schema.",
+  "_comment": "It shows tables used to set a cover art for releases and release groups. The main tables for these two entity types are highlighted. It also shows when tables belong to the 'cover_art_archive' schema.",
```
Comment (reosarevok) [inline docs/database_schema_diagrams/source/cover_art_details.json:3]: The change makes sense based on what it's there - why do we highlight the entity types and not the cover art bits though if those are supposed to be the core of this diagram?

File: docs/database_schema_diagrams/source/entity_network_details.json [hunk 5]
```diff
@@ -1,6 +1,6 @@
 {
-  "tooltip": "Tables specifically related to core entities",
-  "_comment": "It shows tables unique to some entities, that is, beyond aliases, collections, edits, relationships, subscriptions, ratings, tags.",
+  "tooltip": "Tables uniquely related to primary entities",
+  "_comment": "It shows the main tables for primary entity types and tables unique to some of these, that is, beyond aliases, collections, edits, relationships, subscriptions, ratings, tags.",
```
Comment (reosarevok) [inline docs/database_schema_diagrams/source/entity_network_details.json]: _overview talks about "the main tables for primary entities" while this talks about "the main tables for primary entity types". Is that intentional?

File: docs/database_schema_diagrams/source/work_entity_details.json [hunk 6]
```diff
@@ -1,6 +1,6 @@
 {
-  "tooltip": "Tables for the “work” core entity properties",
-  "_comment": "It shows replicated tables used to define any work entity, beyond relationships. It can be used as an example for tables generally used to define any entity of any type. The core entity is highlighted.",
+  "tooltip": "Tables for the “work” entity properties",
```
Comment (reosarevok) [inline docs/database_schema_diagrams/source/work_entity_details.json:2]: Others say "entity type"

File: docs/database_schema_diagrams/source/entity_network_details.json (hunk 3 above)
Comment (yvanzo) [reply docs/database_schema_diagrams/source/entity_network_details.json]: Correct.

File: docs/database_schema_diagrams/source/relationship_overview.json [hunk 7]
```diff
@@ -1,6 +1,6 @@
 {
   "tooltip": "Tables for relationship connections",
-  "_comment": "Show minimal foreign keys needed to define a relationship between two entities. Artist-Work is just an example of relationship type. Main table is highlighted. All other tables are shortened to focus on foreign key constraints.",
+  "_comment": "It shows minimal foreign keys needed to define a relationship between two entities. Artist-Work is just an example of relationship type. The main table is highlighted. All other tables are shortened to focus on foreign key constraints.",
```
Comment (mwiencek) [inline docs/database_schema_diagrams/source/relationship_overview.json:3]:
```suggestion
  "_comment": "It shows minimal foreign keys needed to define a relationship between two entities. Artist-Work is just an example of a relationship type. The main table is highlighted. All other tables are shortened to focus on foreign key constraints.",
```

File: docs/database_schema_diagrams/source/entity_network_details.json (hunk 5 above)
Comment (yvanzo) [reply docs/database_schema_diagrams/source/entity_network_details.json]: No, it’s been addressed in the meantime.

File: INSTALL.md (hunk 2 above)
Comment (reosarevok) [reply INSTALL.md:337]: Does a span show differently? If so, ok :)

File: docs/database_schema_diagrams/source/entity_network_details.json [hunk 8]
```diff
@@ -1,6 +1,6 @@
 {
-  "tooltip": "Tables specifically related to core entities",
-  "_comment": "It shows tables unique to some entities; That is beyond alias, collection, disambiguation comment, edit, link, subscription, rating, tag.",
+  "tooltip": "Tables uniquely related to primary entities",
```
Comment (mwiencek) [inline docs/database_schema_diagrams/source/entity_network_details.json:2]: should that be "primary entity types"?

File: docs/database_schema_diagrams/source/work_entity_details.json (hunk 6 above)
Comment (yvanzo) [reply docs/database_schema_diagrams/source/work_entity_details.json]: Good catch, now fixed in commit https://github.com/metabrainz/musicbrainz-server/pull/2987/commits/2e2bba792d614a75c0e15a36c80aacca6edd53af.

File: INSTALL.md (hunk 2 above)
Comment (yvanzo) [reply INSTALL.md:337]: No, I mean the source Markdown file would be less readable.

File: docs/database_schema_diagrams/source/entity_network_details.json (hunk 8 above)
Comment (yvanzo) [reply docs/database_schema_diagrams/source/entity_network_details.json]: yeah, probably, changed in commit https://github.com/metabrainz/musicbrainz-server/pull/2987/commits/ff41bd6b09f2a563e01c78e2b15e07a598d230d7

File: INSTALL.md (hunk 2 above)
Comment (reosarevok) [reply INSTALL.md:337]: Not sure I agree, but it's also not too important so let's let it be as is :)

**Review by reosarevok — APPROVED** [review]
I think this is fine now :)

**Comment by yvanzo** [comment]
Updated images with new SVG file and summary from `_comment` field.

**Pull request merged**